docker-compose.yml

```diff
@@ -139,6 +141,7 @@ services:
   kafka:
     <<: *restart_policy
     image: "confluentinc/cp-kafka:7.6.1"
+    user: root
```

Comment:
any reason we're using root here?

Reply:
This is from #3496, unrelated to this patch and should be a no-op once the other one is merged.